autopush/tests/test_websocket.py

```diff
@@ -59,6 +59,7 @@
 dummy_chid = uuid.uuid4()
 dummy_chid_str = str(dummy_chid)
 dummy_uaid = uuid.uuid4()
+dummy_uaid_str = uuid.uuid4().hex
```

Comment:
Silly question, but why not convert the dummy_uaid like we do for the chid? (take the `dummy_uaid.hex` value)

If we ever need to really check the string value of dummy_uaid, might make it more consistent.